src/containers/sidebars/featured-place-card/featured-place-card.js

```diff
@@ -27,10 +27,14 @@ function FeaturedPlaceCardContainer(props) {
     description: '',
   });
 
+  const handleClose = () => {
+    browsePage({ type: FEATURED });
+    window.location.reload();
```

Comment:
It can probably be solved in a more elegant way

Reply:
Won't it be enough to reset the `selectedFeaturedPlace`? I don't think we have to browsePage or reload for this